src/lib/i18n.json

```diff
@@ -37,6 +37,20 @@
     "hld-append-var-group-cmd-failed": "HLD Append Variable Group command was not successfully executed.",
     "hld-append-var-group-name-missing": "Variable group name was not provided. Provide variable group.",
 
+    "project-install-lifecycle-pipeline-cmd-failed": "Project install lifecycle pipeline was not successfully executed.",
+    "project-err-github-repo": "GitHub repos are not supported. Repo url: {0} is invalid",
+    "project-err-missing-values": "Value for {0} is missing.",
+    "project-err-init-dependency": "Please run `spk project init` and `spk project cvg` commands before running this command to initialize the project.",
+    "project-err-cvg": "Please run `spk project cvg` command before running this command to create a variable group.",
+    "project-err-repo-url-undefined": "Repo url not defined",
+    "project-err-spk-config-missing": "SPK Config is missing",
+    "project-err-invalid-options": "Invalid option values",
+    "project-err-pipeline-create": "Error occurred during pipeline creation for {0}",
+    "project-err-invalid-build-definition": "Invalid BuildDefinition created, parameter 'id' is missing from {0}",
```

Comment:
can you replace `project-err-...` with `project-install-life-cycle-pipeline-err...`, in this manner, the key in this i18n file will be unique. and it hope to keep the `<command>-<sub-command>-` convention. `project-err-...` is too board.